### PR TITLE
Feature/improve readme

### PR DESCRIPTION
**Pull request by Vivek-H-Rana**
update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+install the oemof-tabular-plugins package.
+
+    pip install oemof_tabular_plugins==0.0.2rc4 numpy==1.26.0 git+https://github.com/oemof/oemof-tabular.git@dev git+https://github.com/sedos-project/oemof.industry.git@saltwater
+
+Possible Errors: 
+1) Please copy and paste the whole statement, there are no line skips.
+2) For Windows users, an error might appear when installing oemof-tabular: 
+    
+    "ERROR: Could not build wheels for cchardet, which is required to install pyproject.toml-based projects"
+3) Guidelines on how to fix this error: https://github.com/twintproject/twint/issues/1407#issuecomment-1141734344
```
Comment (Bachibouzouk) [inline README.md:36]: I would move this part to a troubleshooting section

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+If unsuccessful, the message will say "cbc: not working", in which case an external executable "cbc.exe" file from a 
+successful installation should be included in your path and your system restarted.
+
+Once all of these steps have been completed successfully, you are ready to run scenarios!
+
+## Get started
+
+
+## Installation
```
Comment (Bachibouzouk) [inline README.md:52]: From here this is part the the template, I would leave it out for the moment :)

**Review by Bachibouzouk — COMMENTED** [review]
Thanks @Vivek-H-Rana for providing a first draft of readme

Could you also try follow the readme which is in the `component_library` folder and read the python code there as well?

File: README.md (hunk 2 above)
Comment (Vivek-H-Rana) [reply README.md]: okay understood

File: README.md (hunk 1 above)
Comment (Vivek-H-Rana) [reply README.md]: its done in the next commit

**Comment by Vivek-H-Rana** [comment]
Will try my best to follow and read it

**Comment by Vivek-H-Rana** [comment]
As such there will be no changes more from my side,unless necessary, so you may close the pull request.

**Pull request merged**